src/components/features/login/Login.js

```diff
@@ -0,0 +1,99 @@
+/* eslint-disable camelcase */
+/* eslint-disable react/no-unused-prop-types */
+/* eslint-disable react/prefer-stateless-function */
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { Link, withRouter } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import { toast } from 'react-toastify';
+import { compose } from 'redux';
+import Input from '../../global/Input';
+import Button from '../../global/Button';
+import userLogin from '../../../actions/authActions';
+import storeToken from '../../../helpers/authHelper';
+
+export class Login extends Component {
+  // eslint-disable-next-line react/state-in-constructor
+  state = {
+    email: '',
+    password: '',
+    isLoading: false,
+  }
+
+  UNSAFE_componentWillReceiveProps(nextProps) {
+    const {
+      data, error, status, history,
+    } = nextProps;
+    switch (status) {
+      case 'Success':
+        storeToken(data.data.token);
+        toast.success(data.message);
+        history.push('/');
+        this.setState({ isLoading: false });
+        break;
+      case 'Failure':
+        toast.error(error.data.message);
+        this.setState({ isLoading: false });
+        break;
+      default:
+        break;
+    }
+  }
+
+  submitForm = (e) => {
+    e.preventDefault();
+    const { email, password } = this.state;
+    const credentials = {
+      email,
+      password,
+    };
+    const { userLogin: login } = this.props;
+    this.setState((state) => ({ ...state, isLoading: true }));
+
+    login(credentials);
+  };
+
+  handleChange = (e) => {
+    e.preventDefault();
+    const { name, value } = e.target;
+    this.setState({
+      [name]: value,
+    });
+  };
+
+
+  render() {
+    const { isLoading } = this.state;
+    return (
+      <div className="login-form">
+        <form onSubmit={this.submitForm}>
+        <Input type="email" name="email" placeholder="Email..." onChange={this.handleChange} required />
+        <Input type="password" name="password" placeholder="Password..." onChange={this.handleChange} required />
```

Comment:
Pass in the value attributes here because it is set as isRequired props in the Input component and complaining.